src/legacy/index.html

```diff
@@ -5,13 +5,13 @@
     <title>Florence</title>
 
     <!--Allows user to save link to Florence to desktop -->
-    <link rel="manifest" href="/florence/dist/manifest.json">a
+    <link rel="manifest" href="/florence/dist/manifest.json">
 
     <link href="https://cdn.ons.gov.uk/assets/florence/jquery-tag-it/jquery.tagit.css" rel="stylesheet" type="text/css">
     <link href="https://cdn.ons.gov.uk/assets/florence/jquery-tag-it/tagit.ui-florence.css" rel="stylesheet" type="text/css">
     <link href="https://cdn.ons.gov.uk/assets/florence/pagedown/pagedown.css" rel="stylesheet" type="text/css">
     <link href="https://cdn.ons.gov.uk/vendor/sweetalert/1.0.0/sweetalert.css" rel="stylesheet" type="text/css">
-    <link href="https://cdn.jsdelivr.net/npm/select2@4.1.0-rc.0/dist/css/select2.min.css" rel="stylesheet" />
+    <link href="/florence/src/legacy/css/third-party/select2.min.css" rel="stylesheet" />
```

Comment:
You will want to add this to https://github.com/ONSdigital/cdn.ons.gov.uk-vendor/tree/master/vendor I believe. Then change the URL to point to the CDN